ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/MainActivity.kt

```diff
@@ -162,10 +198,27 @@ fun WearApp(watchRepository: WatchRepository) {
                         navBackStackEntry.arguments?.getInt(WATCH_ID_NAV_ARGUMENT)!!
                     WatchDetailScreen(
                         id = watchId,
+                        scrollState = watchDetailScrollState,
                         watchRepository = watchRepository
                     )
                 }
             }
         }
     }
 }
+
+/**
+ * Used for the 'in' comparisons in the Scaffold section for determining which screen is visible
+ * and what to show (vignette and position indicator). Note, 'in' is translated to the operator
+ * 'contains' fun in Kotlin.
+ */
+operator fun String?.contains(substring: String): Boolean {
```

Comment:
This seems a little strange to me, why not just use equals?

Reply:
Yep, that's why.

I used the `contains` based on the code in Jetsnack, but, yeah, I think no extension fun with just `startsWith` is probably clearer (especially for folks who aren't used to the `in` + `operator contains` convention).

Changed, PTAL.

Reply:
Looks good!